src/components/user/ImageUpload.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import axiosInstance from '../../apis/axiosClient';
+import { useAuth } from '@akosasante/react-auth-context';
+
+const ImageUpload = () => {
+  const [selectedFile, setSelectedFile] = useState(null);
+  const [uploadStatus, setUploadStatus] = useState('');
+  const { user } = useAuth(); //use user.user.<whatever field we want> to access it properly
+
+  const handleFileChange = (event) => {
+    // Update the state with the selected file
+    setSelectedFile(event.target.files[0]);
+  };
+
+  const handleUpload = async () => {
+    // Check if a file is selected
+    if (!selectedFile) {
+      alert('Please select a file to upload.');
+      return;
+    }
+
+    // Create a FormData object to send the file
+    const formData = new FormData();
+    formData.append('userImage', selectedFile);
+
+    try {
+      // Send a POST request to the server with the file
+      const response = await axiosInstance.post(
+        '/users/uploadUserImage',
+        formData
+      );
+
+      if (response.ok) {
```

Comment:
Axios doesn't set a `ok` variable on its response values like `fetch` does. In axios, if the response status code is not in the 200s, then it will reject the promise (which for async/await, means that an error will be raised that you can handle in the `catch` part of this try/catch). So we don't need to check if the response is ok here. 

Currently, this is causing the error message to be shown because `response.ok` is undefined, so this if/else, will go to the `else` part ofthe statement